_data/SamReeve96/SamReeve96.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Samuel Reeve
+institution: University of Portsmouth
+profile_pic: SamuelReeve.JPG
+quote: The Best Thing To Do Is Stop Trying To Figure Out Where You're Going And Just Enjoy Where You’re At
```

Comment:
Change quote ’ from non-ascii to ascii ' character.

Reply:
Done, thanks for the link to the validator, can't believe it got past me 😅